text/0046-404-405-http-error-code.md

```diff
@@ -0,0 +1,121 @@
+- Title: 404/405 Http Error Codes
+- Start Date: 2021-06-07
+- Specification PR: [#46](https://github.com/meilisearch/specifications/pulls/46)
+- MeiliSearch Tracking-issues:
+
+
+# 404/405 Http Error Codes
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+The purpose of this file is to specify the behavior of the API about the http code to return in case a resource does not exists or the http method used is not allowed on the resource.
```

Comment:
```suggestion
The purpose of this file is to specify the behavior of the API about the HTTP code to return in case a resource does not exist or the HTTP method used is not allowed on the resource.
```